packages/app/src/flame/valibot/schemaUtil.ts

```diff
@@ -0,0 +1,11 @@
+import { f32, struct } from 'typegpu/data'
+
+export function schemaToF32Struct<T extends Record<string, unknown>>(
```

Comment:
Potentially = writing small lib to go from valibot to typegpu schema or reverse, for now, just a utility function for what is used by most parametric variations